packages/transfer-app/src/Transfer.tsx

```diff
@@ -23,6 +23,9 @@ export class Transfer extends React.PureComponent<Props> {
 
         <Switch>
           <Route component={SentBalance} path='/transfer/:currentAccount/sent'></Route>
+          <Route exact path="/transfer/:currentAccount/" render={({match: {params: {currentAccount}}}) => (
+              <Redirect to={`/transfer/${currentAccount}/${currentAccount}`} />
```

Comment:
suggestion: perhaps we should make the default "to" address the zero-th address from keyring addresses?

Reply:
yes, it'd be better; my idea was to make the default recipient address be the first address from the keyring/addresses that is not currentAccount (and if there is only one account, then let it be currentAccount), however I thought this complexifie the code maybe a bit needlessly, i.e. having to call the observables etc.

but I think you're right, I'll do this